nx/include/switch/services/hwopus.h

```diff
@@ -18,8 +18,10 @@ typedef struct {
 /// This structure is the start of opusin for \ref hwopusDecodeInterleaved, with the actual opus packet following this.
 /// These fields are big-endian.
 typedef struct {
-    u32 size; ///< Size of the packet following this header.
-    u32 unk;  ///< Unknown, can be left at zero.
+    /// Size of the packet following this header.
```

Comment:
Keep comments on the same lines like before, also keep " can be left at zero."

Reply:
Done.